src/Modal/Modal.js

```diff
@@ -185,7 +175,7 @@ Modal.Header = ({ title, children, showClose = true }) => {
 
           {showClose && (
             <Box ml="auto">
-              <Icon name="close" color="greyDark" style={{ cursor: 'pointer' }} onClick={handleClose} />
+              <Icon name="close" color="greyDark" style={{ cursor: 'pointer' }} size={16} onClick={handleClose} />
```

Comment:
I think this may need to be bumped up to 24, looking a little small 

![Screen Shot 2019-08-21 at 2 38 39 PM](https://user-images.githubusercontent.com/14184138/63470415-7bb5fd00-c421-11e9-9017-b279e9cce3b2.png)